### ISSUE TITLE
WETH logo url is busted in the tokenlist

### ISSUE THREAD

**Issue by dkent600** [PrimeDAO/prime-launch-dapp#409]

**Comment by dkent600** [comment]
This is what etherscan is using, but Kolektivo is preferring to use logos from IPFS. 

https://etherscan.io/token/images/weth_28.png

**Comment by dkent600** [comment]
just tested, seems to be working ok now.

**Comment by BartuCulha** [comment]
Logo is busted inside the dropdown menu but it's visible in other fields.
![Launch Details _ (7)](https://user-images.githubusercontent.com/24368665/140726291-3c811ed3-5ea6-481c-92d9-6959e763e8d1.png)

**Comment by BartuCulha** [comment]
Is it possible to use any other URL for the Logo? Current one is smaller than the rest. @dkent600 
![LBP Summary _ Re](https://user-images.githubusercontent.com/24368665/140726592-09059e74-6897-4fe7-a9f8-b016fe3d6c40.png)

**Comment by dkent600** [comment]
> Is it possible to use any other URL for the Logo? Current one is smaller than the rest. 

I guess we we have to talk to kolektivo about that.  @renc-curvelabs   @BartuCulha

**Comment by renc-curvelabs** [comment]
@BartuCulha can you confirm if this is still an issue

**Comment by dkent600** [comment]
> @BartuCulha can you confirm if this is still an issue

@renc-curvelabs The icon is present.  It just doesn't look great.

**Comment by BartuCulha** [comment]
yep it's working shall we move this card to won't work?  @dkent600

**Comment by dkent600** [comment]
@BartuCulha move to accepted
